docs/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -151,117 +150,140 @@ This guide shows how to get a :ref:`batch <reference__core_concepts__batches>` o
+              from great_expectations.core import ExpectationSuite
+              from great_expectations.core.batch import RuntimeBatchRequest
+              from great_expectations.validator.validator import Validator
 
-            Create an in-code Data Context using these instructions: :ref:`How to instantiate a Data Context without a yml file <how_to_guides__configuring_data_contexts__how_to_instantiate_a_data_context_without_a_yml_file>`
+              context: DataContext = ge.get_context()
+
+          Create an in-code Data Context using these instructions: :ref:`How to instantiate a Data Context without a yml file <how_to_guides__configuring_data_contexts__how_to_instantiate_a_data_context_without_a_yml_file>`
 
 
         1. Configure a Datasource
 
-            Configure a :ref:`Datasource <reference__core_concepts__datasources>` using the :ref:`RuntimeDataConnector <reference__core_concepts__datasources>` to connect to your DataFrame. Since we are reading a Pandas DataFrame, we use the ``PandasExecutionEngine``. You can use ``batch_identifiers`` to define what data you are able to attach as additional metadata to your DataFrame using the ``data_connector_query`` parameter (shown in step 3).
+          Configure a :ref:`Datasource <reference__core_concepts__datasources>` using the :ref:`RuntimeDataConnector <reference__core_concepts__datasources>` to connect to your SQL database. Since we are using a SQL database, we use the ``SqlAlchemyExecutionEngine``. You can use ``batch_identifiers`` to define what data you are able to attach as additional metadata to your Batch using the ``batch_identifiers`` parameter (shown in step 3).
+
+          By default, the SqlAlchemy Execution Engine will create a temporary table using a given query (provided in step 3). This has a performance advantage when creating and working with a Batch because the query will only be executed once (when the temporary table is created). If you would like to override this default behavior (for example, if you do not have permissions to create a temporary table), you may do so by setting ``create_temp_table`` to ``False`` in the Execution Engine configuration. You may also override the default behavior at runtime, on a case-by-case basis via the ``batch_spec_passthrough`` argument of a Runtime Batch Request (see step 3 for details).
+
+          .. code-block:: yaml
+
+              insert_your_sqlalchemy_datasource_name_here:
+                class_name: Datasource
+                module_name: great_expectations.datasource
+                execution_engine:
+                  class_name: SqlAlchemyExecutionEngine
+                  module_name: great_expectations.execution_engine
+                  connection_string: sqlite:///my_db_file # Insert your SqlAlchemy connection string here
+                  # create_temp_table is optional and defaults to True - override this behavior here
+                  create_temp_table: False
```

Comment:
really knitpicky but if the comment was `you may override the behavior here`, then it would be more consistent with the statement above.